lubw_downloader/Dockerfile

```diff
@@ -0,0 +1,23 @@
+-FROM alpine:latest 
+
+# install dependencies
+RUN apk update && \
+    apk del sed && \
+    apk add \
+    bash \
+    curl \
+    postgresql-client \
+    sed
+
+# copy and credentials
+COPY resources/ . 
+RUN mkdir -p /data/xml_data /data/temp
+RUN chmod -R 775 /data /scripts 
+
+# set time zone 
+RUN ln -snf /usr/share/zoneinfo/Europe/Berlin /etc/localtime && echo Europe/Berlin > /etc/timezone
+
+# add to crontab
+RUN crontab /cron/tab.txt
```

Comment:
Are you sure `cron/tab.txt` is the one of the `resources/` folder? Not sure what the PWD is while copying.